website/docs/guides/mesh-qs.md

```diff
@@ -83,6 +83,20 @@ To [create](/docs/cloud/about-cloud-setup) a new project in dbt Cloud:
     - New git repo
     - One or more [environments](/docs/deploy/deploy-environments) (such as development, deployment)
 
+### Create a production environment
+In dbt Cloud, each project can have one designated deployment environment, which serves as its production environment. You must set up a [production deployment environment](/docs/deploy/deploy-environments#set-as-production-environment) for each project you want to explore. This enables you to leverage dbt Explorer in the [later steps](https://docs.getdbt.com/guides/mesh-qs?step=5#create-and-run-a-dbt-cloud-job) of this guide.
```

Comment:
@jtcohen6 since we're adding this new step, should we go ahead and add staging to this as well, since it's a blocker for mesh adoption amongst some clients?

Reply:
@matthewshaver Really good idea! Think we could add that in here quickly? Basically, customers need a Production and/or Staging environment

Reply:
do you mean add 'Staging' as an important developer workflow to better test data without affecting production? or is it now that in order to leverage explorer, they can have a production or staging environment enabled?

Reply:
@kbrock91 made the very good point that we should keep this quick (quickstart) rather than exhaustive (docs). So at most I'd expect an extra word in here like:
```suggestion
In dbt Cloud, each project can have one deployment environment designated as "Production.". You must set up a ["Production" or "Staging" deployment environment](/docs/deploy/deploy-environments) for each project you want to "mesh" together. This enables you to leverage dbt Explorer in the [later steps](https://docs.getdbt.com/guides/mesh-qs?step=5#create-and-run-a-dbt-cloud-job) of this guide.
```
Just a suggestion, not saying those words exactly

Reply:
i love it!